packages/js-core/src/lib/initialize.ts

```
@@ -122,38 +160,32 @@
+        return err(res.error);
+      }
+      updatedAttributes = res.value;
+    } else {
+      updatedAttributes = { ...configInput.attributes };
     }
-    updatedAttributes = res.value;
   }
 
   if (
     existingConfig &&
     existingConfig.environmentState &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)